src/test/kotlin/org/rust/lang/core/completion/RsFragmentSpecifierCompletionTest.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.completion
+
+import org.intellij.lang.annotations.Language
+import org.rust.lang.core.macros.FragmentKind
+
+class RsFragmentSpecifierCompletionTest : RsCompletionTestBase() {
+    fun `test fragment specifiers`() = doTest("""
+        macro_rules! foo {
+            (${"$"}l:/*caret*/) => {}
```

Comment:
```suggestion
            ($ l:/*caret*/) => {}
```
(note space after `$`)